datalab/datalab_session/tests/test_utils.py

```diff
@@ -0,0 +1,92 @@
+import pathlib as pl
+from hashlib import md5
+
+from django.test import TestCase
+
+from datalab.datalab_session.file_utils import *
+from datalab.datalab_session.s3_utils import *
+
+# extending the TestCase class to include a custom assertions for file operations
+class FileExtendedTestCase(TestCase):
+    def assertIsFile(self, path):
+        if not pl.Path(path).resolve().is_file():
+            raise AssertionError("File does not exist: %s" % str(path))
+    
+    def assertFilesEqual(self, image_1: str, image_2: str):
+        with open(image_1, 'rb') as file_1, open(image_2, 'rb') as file_2:
+            self.assertEqual(md5(file_1.read()).hexdigest(), md5(file_2.read()).hexdigest())
+
+class FileUtilsTestClass(FileExtendedTestCase):
+
+  test_fits = 'datalab/datalab_session/tests/test_files/file_utils/fits_1.fits.fz'
+  test_tif = 'datalab/datalab_session/tests/test_files/file_utils/tif_1.tif'
+  test_small_jpg = 'datalab/datalab_session/tests/test_files/file_utils/jpg_small_1.jpg'
```

Comment:
Again, appending `_path` to the names of these variable would make it quite clear they're paths later on in the test